pkgs/games/exult/default.nix

```diff
@@ -12,35 +12,30 @@ let
 in
 
 stdenv.mkDerivation rec {
-  name = "exult-1.4.9rc1";
+  name = "exult-1.5.0git";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/exult/${name}.tar.gz";
-    sha256 = "0a03a2l3ji6h48n106d4w55l8v6lni1axniafnvvv5c5n3nz5bgd";
+  src = fetchFromGitHub {
+    owner = "exult";
+    repo = "exult";
+    rev = "b727abfffc08a54e528bc1194d0de8562d18d74e";
+    sha256 = "1nvxpsycn7r2yiqlyjq8rx9wy2ywc8lnqivh6jzdh9rxryq05j94";
   };
 
-  configureFlags = [ "--disable-tools" ];
+  preConfigure = "./autogen.sh";
```

Comment:
Add `autoreconfHook` (instead of `automake` and `autoconf` and `libtool`) to `nativeBuildInputs`.